### ISSUE TITLE
"command not found" on install

### ISSUE THREAD

**Issue by NickDouglas** [jbrew/pt-voicebox#18]
Hi! I don't know much about Terminal or coding, so this might be a dumb question.

When I follow step 5 of "Running the Program" (by entering "pip install -r requirements.txt" in terminal from the pt-voicebox folder, I get the error message "command not found". Do I need to install something else to enable that command?
![screen shot 2016-11-09 at 2 21 43 pm](https://cloud.githubusercontent.com/assets/23365435/20151394/fdb3b87a-a687-11e6-973d-99a0da8f69a7.png)

**Comment by pennetti** [comment]
Hey Nick! 

Thanks so much for submitting this issue, not a dumb question at all - this is definitely not an intuitive process (especially if you're opening the terminal for the first time!). I forgot that `pip` is not installed by default (oops!), could you try running `sudo easy_install pip` (should ask for your password if you have one) and let me know if that fixes it? If that works I'll update the instructions to include this step so nobody else runs into this.

**Comment by NickDouglas** [comment]
Thank you! The installation seemed to work fine. Now I'm up to step 7 and I got this:
![screen shot 2016-12-01 at 11 11 34 am](https://cloud.githubusercontent.com/assets/23365435/20801476/14c23bc8-b7b7-11e6-8a9b-1c9e7cf61049.png)

I'm not sure what to do next. I also tried running voicebox from Finder:
![screen shot 2016-12-01 at 11 13 36 am](https://cloud.githubusercontent.com/assets/23365435/20801522/3e37b708-b7b7-11e6-92b0-7f656a0cb0d7.png)

So with either method, I'm not seeing any instructions.

Again, thank you for your patience with a layman!

**Comment by born-slippery** [comment]
Hi Nick, having exactly the same issue with the ImportError. 
Have you found a solution?

Thanks!

**Comment by misuba** [comment]
Yep, I'm seeing this too. Is it to do with command-line Mac OS dev tools being installed or not, maybe?

**Comment by pennetti** [comment]
Hey @NickDouglas, @born-slippery, and @misuba,

It looks like this is caused by an old version of the `six` package. Could you try upgrading it to see if that fixes your issue? The command is `sudo easy_install --upgrade six`. Try running from the root directory, so inside `pt-voicebox` run `bin/voicebox`. Also, in the future could you please paste the text from the terminal instead of taking a screenshot? 

Sorry about the slow responses here, I'll try to keep up with these issues. Thanks so much for checking out the project!
